docs/contributing.md

```diff
@@ -82,6 +82,14 @@ existing component, we require an architecture review from  multiple
 stakeholders, which we do with [Design Docs](design_docs.md), see the
 [process here](design_docs.md#process).
 
+## Deprecating and removing commands
+
+Before removing a until now required argument, or making a optional argument
+required or even a user-visible command, make yourself familiar with our
```

Comment:
"or even a user-visible command" seems out of place.

Reply:
rephrased